content/docs/higher-order-components.md

```diff
@@ -1,56 +1,55 @@
 ---
 id: higher-order-components
-title: Higher-Order Components
+title: Componentes de Ordem Superior
 permalink: docs/higher-order-components.html
 ---
 
-A higher-order component (HOC) is an advanced technique in React for reusing component logic. HOCs are not part of the React API, per se. They are a pattern that emerges from React's compositional nature.
+Um componente de ordem superior (HOC, do inglês Higher-Order Component) é uma técnica avançada no React para reutilizar a lógica de um component. HOCs não são parte da API do React por si só. Eles pertencem a um padrão que surgiu da própria natureza de composição do React.
```

Comment:
I still used `HOC` as the abbreviation, though I'm not sure if I should have used `COS` instead.

Reply:
Dessa maneira, aparecendo no começo do texto com o inglês do lado é perfeitamente possível. 

Por mim pode manter assim... eu acho bem desagradável quando traduzem siglas já conhecidas, vira outra coisa... é que nem ler um texto sobre DNA que os caras tratam como ADN, dá um bug mental!